modules/xmpp/SDP.js

```diff
@@ -200,9 +199,9 @@ SDP.prototype.toJingle = function (elem, thecreator) {
 
                 elem.up();
             }
-            if (SDPUtil.find_line(this.media[i], 'a=crypto:', this.session)) {
+            var crypto = SDPUtil.find_lines(this.media[i], 'a=crypto:', this.session);
+            if (crypto.length) {
```

Comment:
Would this be a good candidate for a function extraction in the path for avoiding having functions with hundreds of lines?